### ISSUE TITLE
Make wlpHome not necessary

### ISSUE THREAD

**Issue by hutchig** [OpenLiberty/liberty-arquillian#39]
Enhancement
https://stackoverflow.com/questions/54390667/arquillian-and-open-liberty-requires-existing-installation/54424464#54424464
##### Issue Overview

I'm familiar with Tomcat/TomEE and testing applications using Arquillian. Now were are switching to Open Liberty. I see there is a module for Arquillian using embedded Open Liberty but it seems to require an existing Open Liberty installation whose path is provided in the configuration. This makes it non-portable and therefore unsuitable for automated testing since the installation has to be present at the exact same path. Arquillian and TomEE are self-contained, no installation required. Therefore my question is why this isn't also possible with Open Liberty? And is this planned for the future?

For reference this is how you use Arquillian with TomEE/Tomcat:
```
<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns="http://jboss.org/schema/arquillian"
    xsi:schemaLocation="http://jboss.org/schema/arquillian http://www.jboss.org/schema/arquillian/arquillian_1_0.xsd">
    <container qualifier="tomee" default="true">
        <configuration>
            <property name="httpPort">-1</property>
            <property name="stopPort">-1</property>
            <property name="users">user=pass</property>
        </configuration>
    </container>
</arquillian>
```
As you can see, there is no path to a local installation required to run the tests. The only thing you need to do is a add a couple of Maven dependencies in test scope that pull in TomEE (embedded). If the same would work for Open Liberty that would be great.

#

**Comment by chyt** [comment]
I don't agree with pulling down an Open Liberty runtime if no existing runtime is specified. How does the user manage the configuration of the features/servers on that runtime?

If you're using the Arquillian Liberty managed container, chances are you're probably already using the liberty-maven-plugin as well to manage the Liberty runtime (or at least you should be). In this case we have already built into the liberty-maven-plugin a `configure-arquillian` [goal](https://github.com/WASdev/ci.maven/blob/master/docs/configure-arquillian.md) which automatically generates the proper arquillian.xml based on the pre-existing Liberty runtime configuration. I think this is the proper solution to avoid having to define a wlpHome (or even arquillian.xml at all).

**Comment by hantsy** [comment]
Like Wildfly/JBoss, pick up a **WLP_HOME**(or some other convention name in WLP) firstly, then the **wlpHome** configuration.  I think the simplest case is just adding a `liberty-managed` or `liberty-remote` into dependencies, the tests should work.
